client/src/extension.ts

```diff
@@ -52,10 +50,22 @@ export async function activate (context: ExtensionContext): Promise<void> {
   // Create the language client and start the client.
   client = new LanguageClient('bitbake', 'Language Server Bitbake', serverOptions, clientOptions)
 
+  client.onRequest('custom/verifyConfigurationFileAssociation', async (param) => {
+    if (param.filePath?.endsWith('.conf') === true) {

```

Comment:
There are .conf files that are not related to bitbake. Could we implement a more complete test that tries to guess if this is a bitbake .conf? (some paths are obvious, or some syntax elements?)